GameFighter/Models/Army.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Generic;
+
+namespace GameFighter.Models
+{
+    public class Army
+    {
+        public List<Warrior> ArmyMembers { get; private set; } = new List<Warrior>();
```

Comment:
even with private set any part of code in Army class can reassign the collection. Consider using readonly keyword